roles/openshift_health_checker/test/disk_availability_test.py

```diff
@@ -183,11 +183,12 @@ def test_fails_with_insufficient_disk_space(name, group_names, configured_min, a
         ansible_mounts=ansible_mounts,
     )
 
-    result = DiskAvailability(fake_execute_module, task_vars).run()
+    check = DiskAvailability(fake_execute_module, task_vars)
+    check.run()
```

Comment:
I read the new docstring saying `run` returns a dict but it will be empty and here we ignore the return value altogether. Maybe its less hassle to just get rid of the return value?

From the testing perspective, pure functions returning values tend to be easier to test than side-effects (e.g., attributes), but I'll not go into that discussion :-)

Reply:
I would prefer to either have a return value or record state, not both. I think the long-term direction is to record state because keeping track of a single return value is tedious (passing between methods, etc) and leads to lots of duplicate logic sprinkled around.

I didn't completely refactor to get rid of return values, but would like to do that.